openpype/hosts/blender/plugins/load/load_animation.py

```diff
@@ -61,5 +61,10 @@ def process_asset(
 
         bpy.data.objects.remove(container)
 
-        library = bpy.data.libraries.get(bpy.path.basename(libpath))
+        filepath = bpy.path.basename(libpath)
+        # Blender has a limit of 63 characters for any data name.
+        # If the filepath is longer, it will be truncated.
+        if len(filepath) > 63:
+            filepath = filepath[:63]
+        library = bpy.data.libraries.get(filepath)
```

Comment:
What this actually does? Truncating filepath doesn't seem correct.

Reply:
When loading from a .blend file, the file is added as `library`. The name of the library is the name of the file, but if the name is over 63 characters, it is truncated 😅

![image](https://github.com/ynput/OpenPype/assets/1087869/ba3dcf1b-47de-4d71-9b13-ed35704d59b9)

So, to access that data, I need to truncate the name 🥲

Reply:
BTW is the `filepath` a filepath or filename? From your example it looks like it is filename.